exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -1348,8 +1385,9 @@ func (b *Bitfinex) wsHandleOrder(data []interface{}) {
 		}
 	}
 	if data[13] != nil {
-		if ordStatus, ok := data[13].(string); ok {
-			oStatus, err := order.StringToOrderStatus(ordStatus)
+		if combinedStatus, ok := data[13].(string); ok {
+			statusParts := strings.Split(combinedStatus, " @ ")
```

Comment:
Do you have a reference for this change? The split is fine, I'm just wondering what this is doing here

Reply:
Yes 🎉 
This change is one of the rolled in PRs. See #1311